setup.py

```diff
@@ -23,10 +23,10 @@
     install_requires=[
         "grpcio==1.37.1",
         "grpcio-tools==1.37.1",
-        "ujson>=2.0.0,<=5.1.0",
+        "ujson>=1.35,<=5.1.0",
```

Comment:
ujson1.35 at most only support python3.2, it's not a proper required version for pymilvus, which supports python >=3.6.

So please change it back to  2.0.0, I don't see any reason why we need to change ujson's version here.

![image](https://user-images.githubusercontent.com/51370125/158932072-eb983857-c5e9-42c4-8f6c-2bcc26ffb9f2.png)

https://pypi.org/project/ujson/1.35/

![image](https://user-images.githubusercontent.com/51370125/158932487-6ebb3871-be32-4f17-ba8b-2b443247634b.png)

https://pypi.org/project/ujson/2.0.0/